pages/docs/data-ops/cli.mdx

```diff
@@ -102,6 +102,84 @@ $ DATABASE_CONNECTION_NAME=dev glean preview
 
 Environment variable substitution is not yet supported for Builds that are triggered via a git revision.
 
+## Integration with dbt
```

Comment:
I think dbt should have its own page.  (Aside / feature request: should builds from the UI automatically get the --dbt flag added)